src/domain/primitives/btc.ts

```diff
@@ -0,0 +1,5 @@
+export const toSats = (amount: number): Satoshis => {
+  return {
+    inner: amount,
+  }
+}
```

Comment:
ohh now I see, the name is confusing, with the name I expect some kind of operation and not just a type parser